<?xml version="1.0"?>

<launch>
    <node pkg="tp_rrt_planner" type="tp_rrt_node" name="tp_rrt_node" output="screen" cwd="node">
        <param name="config_file" type="str" value="$(find tp_rrt_planner)/data/eklavya4_bot_config.ini"/>
        <param name="debug" type="bool" value="true"/>
        <param name="nose_following" type="bool" value="false"/>
        <param name="lookahead" type="double" value="3.0"/>
        <param name="use_slam" type="bool" value="false"/>
    </node>
</launch>
